content/pl/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,94 @@
+---
+reviewers:
+- bgrant0607
+- mikedanese
+title: Kubernetes — co to jest?
+content_template: templates/concept
+weight: 10
+card:
+  name: concepts
+  weight: 10
+---
+
+{{% capture overview %}}
+Na tej stronie znajdziesz ogólne informacje o Kubernetesie.
+{{% /capture %}}
+
+{{% capture body %}}
+Kubernetes to przenośna, rozszerzalna platforma oprogramowania *open-source* służąca do zarządzania zadaniami i serwisami uruchamianymi w kontenerach, która umożliwia deklaratywną konfigurację i automatyzację. Ekosystem Kubernetesa jest duży i dynamicznie się rozwija. Serwisy Kubernetesa, wsparcie i narzędzia są szeroko dostępne.
```

Comment:
Nit picking: 
`serwisami` or `usługami`?

Reply:
As per discussion on the Slack, I'd leave it as "serwis".